lib/sarif/bandit_sarif.rb

```diff
@@ -0,0 +1,57 @@
+module Sarif
+  class BanditSarif < BaseSarif
+    BANDIT_URI = 'https://github.com/PyCQA/bandit'.freeze
+
+    def initialize(scan_report)
+      super(scan_report)
+      @uri = BANDIT_URI
+      begin
+        @logs = JSON.parse(@scan_report.log(''))['results']
+      rescue JSON::ParserError
+        @logs = []
+      end
+      @unique_issues = Set.new
+    end
+
+    def parse_issue(issue)
+      key = ""
+      key << issue["filename"].upcase + issue["line_number"].to_s + issue['issue_text']
```

Comment:
Why is `upcase` called on `issues["filename"]`?

Reply:
I've removed the upcase